package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
```

Comment:
The screenshot and live link are broken in the README. Please add the active content or remove the option so it will look more professional.